Dockerfile

```
@@ -0,0 +1,91 @@
+# syntax = docker/dockerfile:experimental
+
+# default value
+ARG BUILD_DIR=unlock-js
```

ok to keep or should we just throw an error if no `BUILD_DIR` is set?